### PR TITLE
Templatize page titles and combine search pages

### PR DESCRIPTION
**Pull request by CyrusJavan**
Example: Page with template name "entity-search" will have title "NetAuth | Entity Search"

### CODE REVIEW

**Comment by the-maldridge** [comment]
Neat solution, I had completely forgotten about using the strings.Title for case transform.  My only problem with this is that there's a lot of duplication.  For example the entity-search and group-search templates are identical except for entity and group terms in each.  What might be better would be to pass in the `.title` value in each render method, and then dedup down the templates that are all the same.

**Comment by the-maldridge** [comment]
Also please take a detour through this article so that your commit messages will match those use elsewhere in the codebase: https://chris.beams.io/posts/git-commit/

**Comment by CyrusJavan** [comment]
> My only problem with this is that there's a lot of duplication. For example the entity-search and group-search templates are identical except for entity and group terms in each. What might be better would be to pass in the `.title` value in each render method, and then dedup down the templates that are all the same.

Not sure I understand, do you mean it might be better if all the *-search pages had `.title` of "Search" ? Also my first approach was to pass `.title` each time Render was called, but this solution seemed more DRY.

**Comment by the-maldridge** [comment]
If you diff `tpl/pages/entity-search/page.tpl` and `tpl/pages/group-search/page.tpl` they're almost identical.  It might be more appropriate to have a `tpl/pages/search/page.tpl` and use that for both and just pass in whether its for entity or for group.

I'm mainly concerned about maintaining a bunch of templates that are almost identical, though I think this entire app is going to max out at fewer than 40 templates.

**Comment by CyrusJavan** [comment]
Maybe this should be two separate PRs. I maintained the setting titles based on template name. Also combined the *-search templates into one.

File: tpl/pages/search/page.tpl [hunk 1]
```diff
@@ -29,12 +29,22 @@
...
-      <td>{{ $element.Number }}</td>
-      <td><a href="/entity/info/{{ $element.ID }}">{{ $element.ID }}</a></td>
-      <td>{{ $element.Meta.DisplayName }}</td>
-    </tr>
+    {{ if eq .name "Entity" }}
+      {{ range $index, $element := .result }}
+      <tr>
+        <td>{{ $element.Number }}</td>
+        <td><a href="/entity/info/{{ $element.ID }}">{{ $element.ID }}</a></td>
+        <td>{{ $element.Meta.DisplayName }}</td>
```
Comment (the-maldridge) [inline tpl/pages/search/page.tpl]: This line introduces a whitespace error.  Please make sure you have trimmed whitespace from the end of your lines (this is probably an editor setting).

File: tpl/pages/search/page.tpl [hunk 2]
```diff
@@ -29,12 +29,22 @@
...
+        <td><a href="/entity/info/{{ $element.ID }}">{{ $element.ID }}</a></td>
+        <td>{{ $element.Meta.DisplayName }}</td>  
+      </tr>
+      {{ end }}
+    {{ else }}
+      {{ range $index, $element := .result }}
+      <tr>
+        <td>{{ $element.Number }}</td>
+        <td><a href="/group/info/{{ $element.Name }}">{{ $element.Name }}</a></td>
+        <td>{{ $element.DisplayName }}</td>
```
Comment (the-maldridge) [inline tpl/pages/search/page.tpl]: This line introduces a whitespace error.  Please make sure you have trimmed whitespace from the end of your lines (this is probably an editor setting).

**Pull request closed without merging**